from .base_exceptions import ServiceError


class DuplicateEstimateError(ServiceError):
    pass